### PR TITLE
Feature/guillaume update

### PR DESCRIPTION
**Pull request by sanguinius69**

### CODE REVIEW

File: docs/languages.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+However you will find a common behavior:
+
+* Some are using multi-stage build with a base image (with all the dependencies) and a lighter runtime image to run the app
+* You need to export your compiled code or source files in the folder named `app`
+* You will need to include a complementary file (depending on your language) to allow dependencies to be downloaded: like a `Gemfile` for `Ruby` or `requirements.txt` for `Python` with `pip`)
+
+* expose the app on port `80`
+* if the hostname is required by the framework/lib then it has to be set to `0.0.0.0`
+* we would recommend to keep the same `ENTRYPOINT` as the example but depending on the language you can freely modify it
```
Comment (Disturbing) [inline docs/languages.md]: Saw some do not allow you to modify it? (php below)?.

File: docs/languages.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+EXPOSE 80
+
+ENTRYPOINT ["npm","run","start"]
+```
+
+However you will find a common behavior:
+
+* Some are using multi-stage build with a base image (with all the dependencies) and a lighter runtime image to run the app
+* You need to export your compiled code or source files in the folder named `app`
```
Comment (Disturbing) [inline docs/languages.md]: Maybe emphasise that you can use any build system or tool you wish, the below is just examples, but the key is the output must be in /app (give an example of maven vs grade or something that shows this as  PRO tip kind of thing.)

File: docs/workspaces.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+title: Workspaces
+---
+
+## Overview
+
+Workspaces are here to allow you to collaborate with people in your team.
+Completely segreggated spaces will allow you to share kintoapps, kintoblocks and manage them in the same environment.
```
Comment (Disturbing) [inline docs/workspaces.md]: Dont use the word environment, confusing with our env feature.

File: docs/workspaces.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+You can invite people by email to collaborate with you on any project you may have in your workspace
```
Comment (Disturbing) [inline docs/workspaces.md]: Note that today we do not have email confirmations nor any emails being sent for invitations, required that the user has to be registered and a validated email today in order to be successfully added.

File: docs/workspaces.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+### Coming Soon:
+
+* Revoking Access to a Workspace
+
+* Leaving a Workspace
+
+* Deleting a Workspace
+
+These feature are not ready yet but you can follow these feature [here](https://github.com/kintohub/backlog/issues/13) on our [GitHub backlog](https://github.com/kintohub/backlog), where you can comment on any open issues, or add any issues you would like us to tackle
```
Comment (Disturbing) [inline docs/workspaces.md]: Email notifications, Pending invitation, Resend invitation.

File: docs/languages.md (hunk 1 above)
Comment (sanguinius69) [reply docs/languages.md]: Yup exactly

**Pull request merged**